#166841009 Implement auth validation (AUTH)

codeBlock-1984 opened the pull request:
#### What does this PR do?
- Implements validation for signup and signin routes

#### Description of Task to be completed?
-  Install express-validator
- Create validation middleware
- Write tests for validation

#### How should this be manually tested?
- 

#### Any background context you want to provide?
- 

#### What are the relevant pivotal tracker stories?
- [#166841009](https://www.pivotaltracker.com/story/show/166841009)

#### Screenshots (if appropriate)
-

#### Questions:
-

Alpha1202 reviewed (approved):
@codeBlock-1984 You did a great job by adding the changes you were asked to add. Like the jsdoc you added in your validations. This looks good to me. Nice Job

chokonaira committed:
feat(auth):implement auth validation

- Install express-validator
- Create validation middleware
- Write tests for validation

[Delivers #166841009]
